apps/ejabberd/src/mod_muc_admin.erl

```diff
@@ -101,7 +116,8 @@ create_instant_room(Host, Name, Owner, Nick) ->
     %% Send presence to create a room.
     ejabberd_router:route(OwnerJID, UserRoomJID, presence()),
```

Comment:
That's avery good question @ludwikbukowski. The API call would be better as it probably gives some result if the creation succeeded or not. There are some other places like this which we should re-write to use module's API.